src/sagemaker/utils.py

```diff
@@ -649,4 +651,61 @@ def _module_import_error(py_module, feature, extras):
     return error_msg.format(py_module, feature, extras)
 
 
+class NotebooksDataConfig(abc.ABC):
```

Comment:
can we generalize this as we can use it to get any config not  specifically Example notebooks

Reply:
Sure